build/wd_rust_binary.bzl

```diff
@@ -82,4 +82,5 @@ def wd_rust_binary(
             "@//build/config:no_build": ["@platforms//:incompatible"],
             "//conditions:default": [],
         }),
+        size = "large",
```

Comment:
We should not be unconditionally marking all rust tests as large

Reply:
Agreed. Updating now. Thanks for the review Felix